wazuh/wazuh-kibana-app
# [FIX] Index pattern selector doesn't display the ignored index patterns

### Description
This PR fixes the index patterns displayed on the index pattern selector.

### Changes
- Fix the ignored index patterns on the index pattern selector.
- Removed wrong behavior when selecting an input on Configuration > Setting and appears as an unsaved setting.

### Test
1. Create or be sure you have at least 2 index patterns and the index pattern selector is enabled.
2. Go to `Configuration` > `Settings`
3. Edit the `ip.ignore` app setting adding some of the index patterns you have and save the configuration.
4. The ignored index patterns should disappear from the selector. Take into account the index pattern selector could disappear if you only have a valid index pattern.

Closes #3338
